README.adoc

```diff
@@ -74,6 +74,12 @@ To dump the state of the wallet in `~/bitcoinj/bitcoinj-test.wallet` with the te
 
 NOTE: These instructions are for macOS/Linux, for Windows use the `wallettool/build/install/wallet-tool/bin/wallet-tool.bat` batch file with the equivalent Windows command-line commands and options.
 
+### Building the reference build
+
+Our reference build (which is also used for our releases) is running within a container to provide good reproducibility.
+Buildah 1.26+, Podman 4.1+ and Docker (with BuildKit) are supported. We tested many combinations of these host OSes:
+Debian, Ubuntu, MacOS, Windows (WSL); and these architectures: amd64, arm64. For instructions see `build.Containerfile`.
```

Comment:
on macOS you must use Docker because Podman doesn't work and Buildah is Linux-only.

Reply:
I see, thanks. I didn't list all working combinations because I thought it's too much of information. Maybe we should recommend one working solution per OS, but we can defer this until people start using the reference build. I'll mark this as resolved.